src/routes/search.js

```diff
@@ -8,13 +8,13 @@ const BufferDiv = styled.div`
     width: 85%;
 `;
 
-const Search = () => (
+const Search = (props) => (
```

Comment:
src/components/app-container.js did you forget to pass down the data prop to search? Why does search need props?

Reply:
Good catch. Fixing it rn.